### ISSUE TITLE
NTR: Modifier: Heel predominance (Extremities, joints & nails)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7976]
New term request Modifier: Heel predominance (hudgin_181114170355):   UMLS: 0018870

**Comment by pnrobinson** [comment]
@MickeySegal
Do you mean Prominent heal?
Otherwise I do not understand, please clarify.

**Comment by MickeySegal** [comment]
The finding is labeled as a Modifier to indicate that it gives further detail about another finding.  There are similar findings for unilateral, distal, proximal, elbow, shoulder, hip and so forth.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000138
- label: Heel localization
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applies to an abnormality that is situated in (or predominant in) the heel.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7976
- synonyms: Heel predominance(NONE)
